docs/metasploit-framework.wiki/Building-The-Docs-Locally.md

```diff
@@ -0,0 +1,51 @@
+# Building the Docs Locally
+Whilst you can always browse the documentation at https://docs.metasploit.com/, sometimes there are cases where you may not be able to reach this site, such as when you are on a client engagement with limited or no access to the public internet.
+
+In these cases, you may want to build the documentation site locally so that you can still access and search the documentation offline. This guide will explain how to do that.
+
+## Overview
```

Comment:
I'm not a fan of this mostly being copy/pasta from the original docs [README](https://github.com/rapid7/metasploit-framework/tree/ee4e6d1e27e82fe150de776b2a6c813ed7f97b38/docs) - I think this will end being a point of confusion when folk try to improve the documentation in one place but not the other, etc.

Reply:
I think this comes down to a similar point to what we stated in the other PR and really is now a question of where and how do we want to centralize our documentation so that people don't have to search multiple locations to find what they are after.

Reply:
I agree having two copies of the same information will be confusing for users and an additional maintenance burden on us.

How about use `git` to move the original doc which will retain the history, make any edits on top of it in a new commit, and then replace the duplicate sections with a reference to this one?

That would make it something to the effect of:

```markdown
# Metasploit docs site

This folder maintains the docs for https://docs.metasploit.com/ and https://github.com/rapid7/metasploit-framework/wiki

## Building the docs site locally
The docs site can be built locally to be viewed offline. See [Building the Docs Locally][1] for more information.
```

Reply:
Sounds like the best approach, I'll see if I can't go ahead and make these changes.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/17028/commits/40b755d6dae89d86c00f81cb463659100fd7c58f